1D/matlab/wave_no_eb.m

```diff
@@ -18,7 +18,7 @@
 
 % Initial data
 % AG
-
+u=init_cond(x);
```

Comment:
@axdn How about you also add an initialization for the -1 tilmestep. You can wait for @hozumiyu 's routine to compute u_xx